tools/generate-files.js

```diff
@@ -1,9 +1,13 @@
 var generator = require('../dist/sfdx-generator.cjs')
 var path = require('path')
-console.log(generator)
-
 var root = path.resolve(__dirname, './..')
-var generator = new generator.Generator("sfdx", root)
+var generator = new generator.Generator({
```

Comment:
Split the instanciation and the declaration of the default option, like:
```javascript
const defaultGeneratorOptions = {foo: 'bar'};
const generator = new generator.Generator(defaultGeneratorOptions);
 ```

Reply:
In fact, add the defaultOptions into `sfdx-generator.ts` and set the constructor default value to it.

Reply:
Not sure about that one, it's only an example of how to use it. The implementation will be determined by the project that use it. I prefer enforce the user to provide the options.